### DISCUSSION TITLE
Developing formulae or casks

### DISCUSSION THREAD

**Discussion by denisgolius** [Homebrew/discussions#4354] [Writing Formulae/Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 584588afb4ab557bcc0536f2794b4d043c333bb9
Last commit: 4 days ago
Core tap JSON: 24 Mar 05:06 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: octa-core 64-bit zen
Clang: N/A
Git: 2.34.1 => /bin/git
Curl: 7.81.0 => /bin/curl
Kernel: Linux 5.15.0-67-generic x86_64 GNU/Linux
OS: Ubuntu 22.04.2 LTS (jammy)
Host glibc: 2.35
/usr/bin/gcc: 11.3.0
/usr/bin/ruby: 3.0.2
glibc: N/A
gcc@11: N/A
gcc: N/A
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Because of the specifics working VictoriaMetrics it need to set different build pipelines for darwin and linux but I can't find information about it or example which I can use. Can some one point me to such documentation?

**Comment by gromgit** [comment]
Since a `victoriametrics` formula already exists, I assume that by "build pipelines" you're talking about _using_ VictoriaMetrics.

Have you tried checking the [project documentation](https://docs.victoriametrics.com/), and/or asking on the [project GitHub](https://github.com/VictoriaMetrics/VictoriaMetrics)? Either path would almost certainly get you answers faster and more reliably than asking on a _package distribution_ project like Homebrew.

**Reply by denisgolius** [reply]
> Since a `victoriametrics` formula already exists, I assume that by "build pipelines" you're talking about _using_ VictoriaMetrics.
> 
> Have you tried checking the [project documentation](https://docs.victoriametrics.com/), and/or asking on the [project GitHub](https://github.com/VictoriaMetrics/VictoriaMetrics)? Either path would almost certainly get you answers faster and more reliably than asking on a _package distribution_ project like Homebrew.

I'm the author of this formula and that's why I'm asking about building pipelines here. 😉
**FYI:** We have different ways (via run make commands) to produce builds for Darwin and Linux platforms but I don't know or don't understand in how to apply them in brew formula... or this is not possible?
